avm/res/sql/server/main.bicep

```diff
@@ -652,3 +689,17 @@ type auditSettingsType = {
   @description('Optional. Specifies the identifier key of the auditing storage account.')
   storageAccountResourceId: string?
 }
+
+type secretsExportConfigurationType = {
+  @description('Required. The resource ID of the key vault where to store the secrets of this module.')
+  keyVaultResourceId: string
+
+  @description('Optional. Default to the resource group where this account is. The resource group name where the key vault is.')
+  resourceGroupName: string?
```

Comment:
However, in general, maybe we should put any review on hold until the spec is actually finalized. To ask you to make changes towards something that may not be final seem a bit ... wrong. I hope, by Thursday, this will change.